### PR TITLE
Run upgrade of API reviews to update package version in cosmos DB

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Run upgrade of API reviews to update package version in cosmos DB

### CODE REVIEW

File: src/java/apiview-java-processor/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>apiview-java-processor</artifactId>
-  <version>1.29.0</version>
+  <version>1.29.1</version>
```
Comment (maririos) [inline src/java/apiview-java-processor/pom.xml:9]: If this follows our version strategy, I believe this should be `1.30.0` as `x.x.1` suggest there was a bug fix and I don't see anything in this PR that suggest that.

File: src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs [hunk 2]
```diff
@@ -352,23 +352,23 @@ public async Task UpdateReviewBackground(HashSet<string> updateDisabledLanguages
...
+                // If review is updated using Azure DevOps pipeline then batch process update review requests
                 if (languageService.IsReviewGenByPipeline)
                 {
                     await UpdateReviewsUsingPipeline(language, languageService, backgroundBatchProcessCount);
                 }
                 else
                 {
                     var reviews = await _reviewsRepository.GetReviewsAsync(false, language, fetchAllPages: true);
-                    foreach (var review in reviews.Where(r => IsUpdateAvailable(r)))
+                    foreach (var review in reviews.Where(r => IsUpdateAvailable(r) || r.Revisions.Any(rev => string.IsNullOrEmpty(rev.SingleFile?.PackageVersion))))
```
Comment (maririos) [inline src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs:371]: we want to update the review if we got the package version, right?

Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs:371]: This check is to find the eligible reviews to rebuild. We will rebuild review data if either parser version is different or if package version is missing in review metadata.

File: src/java/apiview-java-processor/pom.xml (hunk 1 above)
Comment (praveenkuttappan) [reply src/java/apiview-java-processor/pom.xml]: Done.

**Review by maririos — APPROVED** [review]

**Pull request merged**